test/utils/pickers/describeRangeValidation/testTextFieldKeyboardRangeValidation.tsx

```diff
@@ -22,11 +23,10 @@ export const testTextFieldKeyboardRangeValidation: DescribeRangeValidationTestSu
 ) => {
   const { componentFamily, render, isSingleInput, withDate, withTime, setValue } = getOptions();
 
-  if (componentFamily !== 'field' || !setValue) {
-    return;
-  }
+  describeSkipIf(componentFamily !== 'field' || !setValue)('text field keyboard:', () => {
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    const setValue = getOptions().setValue!;
```

Comment:
Yeah, this is annoying. 🙈 
Have you considered at least playing with variable name to avoid the need for `eslint-disable`?
Something like:
```diff
-const { componentFamily, render, isSingleInput, withDate, withTime, setValue } = getOptions();
+const { componentFamily, render, isSingleInput, withDate, withTime, setValue: inSetValue } = getOptions();
...
-// eslint-disable-next-line @typescript-eslint/no-shadow
-const setValue = getOptions().setValue!;
+const setValue = inSetValue!;

```

Reply:
I don't think it matters much, if you prefer I can change the var names, but it is very minimal change imo.

Reply:
Sure, it's clearly a nitpick, feel free to ignore it. 👍